### PR TITLE
add index path

### PR DESCRIPTION
**Pull request by dustinxie**

### CODE REVIEW

File: config_mainnet.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ chain:
   # producerPrivKey: SET YOUR PRIVATE KEY HERE (e.g., 96f0aa5e8523d6a28dc35c927274be4e931e74eaa720b418735debfcbfe712b8)
   chainDBPath: /var/data/chain.db
   trieDBPath: /var/data/trie.db
+  indexDBPath: /var/data/index.db
```
Comment (yutongp) [inline config_mainnet.yaml]: don't we also need to change testnet.yaml?

Comment (dustinxie) [reply config_mainnet.yaml]: fixed

**Review by Newman1515 — COMMENTED** [review]
Ok

**Review by Newman1515 — COMMENTED** [review]
Ok

**Pull request merged**